moto/iam/models.py

```diff
@@ -1292,6 +1292,23 @@ def update_role(self, role_name, role_description, max_session_duration):
         role.max_session_duration = max_session_duration
         return role
 
+    def put_role_permissions_boundary(self, role_name, permissions_boundary):
+        if permissions_boundary and not self.policy_arn_regex.match(
+            permissions_boundary
+        ):
+            raise RESTError(
```

Comment:
Hi @macnev2013, can you add a unit test to verify that this error is thrown?

Reply:
Hey @bblommers . Thanks for the update.

Looks like we already have a test cases for it.
[https://github.com/spulec/moto/blob/master/tests/test_iam/test_iam.py#L2380-L2386](https://github.com/spulec/moto/blob/master/tests/test_iam/test_iam.py#L2380-L2386)

Reply:
That's for the `create_role`-method though. It's the same logic, but we should still test the negative scenario against `put_role_permissions_boundary`

Reply:
Hey @bblommers . I have added test case for `put_role_permissions_boundary`.
Please let me know if anything else is required.

Reply:
Hey @bblommers. I have added test case for put_role_permissions_boundary.
Please let me know if anything else is required.